### PR TITLE
Update appdata with latest releases

### PR DESCRIPTION
**Pull request by hfiguiere**
Resolves: n/a

Update the appdata appstream file to include newer versions.

*Use "x" letter to fill the checkboxes below like [x]*

- [ ] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/musescore-coding-rules)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [x] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

**Comment by hfiguiere** [comment]
1. metadata is cc-0 it doesn't need CLA - the patch is used right now on Falthub to provide up to data MuseScore
2. git master doesn't build on my system (totally unrelated). But the patch applies and the proper final output is verified within the confine of Flatpak SDK.

File: build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in [hunk 1]
```diff
@@ -82,6 +82,89 @@
...
+        <ul>
+          <li>"Save online" failed in some cases</li>
+          <li>Palettes were incorrectly placed when using multiple HighDPI monitors and scaling</li>
+          <li>Palettes disappeared on Ubuntu 18.04 in some cases</li>
+        </ul>
+      </description>
+    </release>
+    <release date="2019-11-13" version="3.3.1">
+      <url>https://musescore.org/en/3.3.1</url>
+    </release>
```
Comment (Howard-C) [inline build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in]: I think this is probably not needed, 3.3.1 was drawn back and that link doesn't work anymore.

Also please sign the CLA, the link is in the description of the PR ;-)

File: build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in [hunk 2]
```diff
@@ -82,6 +82,89 @@
...
+          <li>Scoreview jumped back to start of score during note input on (auto)save</li>
+          <li>Sticking could not be copy-pasted</li>
+          <li>Extensions could not be installed on macOS Catalina</li>
+        </ul>
+      </description>
+    </release>
+    <release date="2019-11-14" version="3.3.2">
+      <url>https://musescore.org/en/3.3.2</url>
+      <description>
+        <p>Fixes</p>
```
Comment (Howard-C) [inline build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in]: Please also add a colon to be consistent with others.

File: build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in [hunk 3]
```diff
@@ -82,6 +82,89 @@
     <content_attribute id="money-purchasing">mild</content_attribute>
   </content_rating>
   <releases>
+    <release date="2019-12-04" version="3.3.4">
+      <url>https://musescore.org/en/3.3.4</url>
+      <description>
+        <p>Fixes</p>
```
Comment (Howard-C) [inline build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in]: Please also add a colon to be consistent with others.

File: build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in (hunk 2 above)
Comment (hfiguiere) [reply build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in]: done

File: build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in (hunk 1 above)
Comment (hfiguiere) [reply build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in]: done

**Comment by hfiguiere** [comment]
Actually it builds. The error was mine.

**Comment by anatoly-os** [comment]
@hfiguiere you need to sign CLA. I cannot merge the PR until you do that (but I really want to :) ).

**Comment by hfiguiere** [comment]
Sorry but I can't. I'm not a lawyer. So getting the CLA reviewed by a legal counsel prior signing to give you something for free is kind of an unbalanced situation.

I was just trying to be helpful in that situation, but it looks like I can't.

**Comment by MarcSabatella** [comment]
To be clear, you don’t need to be a lawyer nor should there be any need to consult with one to sign this simple document.  It’s very short and written in plain English, and just says that you are in fact giving the code away to MuseScore - that you won’t later change your mind and try to take your permission away.

**Comment by riaanvn** [comment]
hi @hfiguiere 

If @MarcSabatella has not managed to quite convince you: I understand your concern with not just agreeing to something when you do not understand the impact. 

Please see this very comprehensive and rounded article by a GitHub employee (product manager and lawyer) about the CLA, why, why nots, pros/cons:

https://ben.balter.com/2018/01/02/why-you-probably-shouldnt-add-a-cla-to-your-open-source-project/

according to which
"GitHub made that relationship explicit by incorporating an inbound = outbound clause into its Terms of Service, meaning if a pull request is submitted via GitHub, it is submitted under the project’s open source license."
https://help.github.com/en/github/site-policy/github-terms-of-service#6-contributions-under-repository-license

If you don't agree to the MuseScore CLA, you are still bound by GitHub's T&Cs. The only risk is that there might be something in the MuseScore CLA which "tricks" you into handing over rights that you did not hand over as part of accepting the GitHub T&Cs (note: there is not, that I can see)
 
Q: Where does that leave you the contributor?
A: The MuseScore CLA is not asking you for anything that you have not agreed to anyway as per the GitHub T&C. They require it for corporate "Cover Your Ass" (the article's words). 

Analogy: In South Africa when we pay by credit card with a PIN, many merchants are still stuck in the "swipe to pay" mode, which requires you to sign the credit card slip. They ask for the signature even though the slip says "no signature necessary". I end up not fighting this and just sign, even though the PIN process has implied "I agree to the T&Cs", and the signature is completely unnecessary / redundant. 

Q: Where does that leave MuseScore (the project and company)?
A: The current approach of separate CLA may not be necessary or could probably require some rework (e.g. getting rid of it and trusting the rights as established by the GitHub T&Cs), but that requires lawyers on the MuseScore (company) and Ultimate Guitar (parent company). 
(Speculating here): The MuseScore CLA requirement may (a) predate the move from SourceForge to GitHub (2013) or (b) the modification on the GitHub T&C to include an explicit "inbound = outbound" clause, and getting rid of the separate MuseScore CLA requirement may not be a high priority (or a matter of education/familiarisation). 

Disclaimer: not a lawyer (have a law degree from a former life, but not practicing). Speaking as a MuseScore community member, not on behalf of anyone but myself.

**Comment by hfiguiere** [comment]
Feel free to merge this if you still feel like it. But don't wait for me to sign the CLA.

**Comment by anatoly-os** [comment]
@hfiguiere thank you for your valuable contribution! I manually merged the changes.
